subsys/bluetooth/Kconfig

```diff
@@ -119,7 +119,7 @@ if BT_CONN
 config BT_HCI_ACL_FLOW_CONTROL
 	bool "Controller to Host ACL flow control support"
 	# Enable if building a Host-only build
-	default y if !BT_CTLR && !BT_STM32_IPM && !BT_ESP32  && !BT_STM32WBA
+	default y if !BT_CTLR && !BT_STM32_IPM && !BT_ESP32 && !BT_STM32WBA && !BT_STM32WB0
```

Comment:
This is starting to get messy. We should probably instead create a virtual option that the HCI drivers select and then here we only depend on that one virtual option. That said, do you realize that `BT_CTLR` is actually a generic option for describing any local (same memory & CPU) controller? I think we should work toward having all such HCI drivers enable `BT_CTRL` and associated options. Nordic already enables `BT_CTRL` for the upstream open source split link layer, as well for their downstream closed source SoftDevice controller, but no other HCI driver does this at the moment.